public/js/acmeair-common.js

```diff
@@ -38,77 +38,63 @@ function hideWaitDialog() {
 }
 
 function updateLoggedInUserWelcome() {
-	var loggedinuser = dojo.cookie("loggedinuser");
-	if (loggedinuser == null) {
+	const session = dojo.cookie("acmeair-user");
+
+	if (!session) {
 		dojo.byId("loggedinwelcome").innerHTML = '';
-	}
-	else {
-		dojo.byId("loggedinwelcome").innerHTML = 'Welcome Back ' + loggedinuser;
+	} else {
+		const data = session.split(';')
+		dojo.byId("loggedinwelcome").innerHTML = 'Welcome Back ' + data[2];
 	}
 }
 
 function login() {
 	hideLoginDialog();
 	showLoginWaitDialog();
 	
-	var userString = document.getElementById('userId').value;
 	dojo.xhrPost({
```

Comment:
if the async/await es6/7 of #3  extends to the frontend, maybe this code should typically be impacted ?